log_relay.go

```diff
@@ -35,12 +35,13 @@ func (exec *sidecarExecutor) configureLogRelay(containerId string,
 
 	// Loop through the fields we're supposed to pass, and add them from the
 	// Docker labels on this container
-	fields := make(log.Fields, len(exec.config.SendDockerLabels))
+	fields := make(log.Fields, len(exec.config.SendDockerLabels) + 1)
```

Comment:
Maybe explain the `+1` here. `log.Fields` will result with all the Docker labels + the LogHostname as "Hostname" field.